RELEASE_NOTES.md

```diff
@@ -7,6 +7,7 @@ This file contains a high-level description of this package's evolution. Release
 
 ## 4.9.1 - T.B.D.
 
+* [Bug Fix] Fix some errors detected in Follow on to PR 2497. [PR #2497](https://github.com/Unidata/netcdf-c/pull/2497) . See [Github #????](https://github.com/Unidata/netcdf-c/pull/????).
```

Comment:
@DennisHeimbigner can you fill in the `????`, or let me know what the correct number should be for this string?